sdk/synapse/Azure.Analytics.Synapse.Artifacts/tests/TriggerRunClientLiveTests.cs

```diff
@@ -25,13 +25,56 @@ public TriggerRunClientLiveTests(bool isAsync) : base(isAsync)
         {
         }
 
-        private TriggerRunClient CreateClient()
+        private TriggerClient CreateTriggerClient()
+        {
+            return InstrumentClient(new TriggerClient(
+                new Uri(TestEnvironment.EndpointUrl),
+                TestEnvironment.Credential,
+                InstrumentClientOptions(new ArtifactsClientOptions())
+            ));
+        }
+
+        private TriggerRunClient CreateRunClient()
         {
             return InstrumentClient(new TriggerRunClient(
                 new Uri(TestEnvironment.EndpointUrl),
                 TestEnvironment.Credential,
                 InstrumentClientOptions(new ArtifactsClientOptions())
             ));
         }
+
+        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/18079 - Missing or invalid pipeline references for trigger but no obvious place to put pipeline?")]
+        [Test]
+        public async Task TestQueryRuns()
+        {
+            TriggerClient triggerClient = CreateTriggerClient();
+            TriggerRunClient runClient = CreateRunClient ();
+
+            TriggerResource resource = await DisposableTrigger.CreateResource (triggerClient, Recording);
+
+            TriggerStartTriggerOperation startOperation = await triggerClient.StartStartTriggerAsync (resource.Name);
+            await startOperation.WaitAndAssertSuccessfulCompletion();
+
+            TriggerRunsQueryResponse response = await runClient.QueryTriggerRunsByWorkspaceAsync (new RunFilterParameters (DateTimeOffset.MinValue, DateTimeOffset.MaxValue));
+            Assert.GreaterOrEqual (response.Value.Count, 1);
+        }
+
+        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/18079 - Missing or invalid pipeline references for trigger but no obvious place to put pipeline?")]
+        [Test]
+        public async Task TestCancelRerun()
+        {
+            TriggerClient triggerClient = CreateTriggerClient();
+            TriggerRunClient runClient = CreateRunClient ();
+
+            TriggerResource resource = await DisposableTrigger.CreateResource (triggerClient, Recording);
+
+            TriggerStartTriggerOperation startOperation = await triggerClient.StartStartTriggerAsync (resource.Name);
+            await startOperation.WaitAndAssertSuccessfulCompletion();
+
+            TriggerRunsQueryResponse response = await runClient.QueryTriggerRunsByWorkspaceAsync (new RunFilterParameters (DateTimeOffset.MinValue, DateTimeOffset.MaxValue));
+            // Find the active run and cancel (CancelTriggerInstanceAsync)
```

Comment:
nit:  Should we remove these comments?

Reply:
I have the entire test [Ignored], since I can't get the trigger APIs to work. I was just noting what was left uncompleted on the test.

I can remove them, or the entire test, whatever is standard.

Reply:
Sounds like `Ignore` is the way to go here;  I mistakenly assumed this was here because "it's something that we sometimes run locally but don't want in the CI runs."    Seems like it's more a case of iterating to try and get things working.

Reply:
Exactly!